### DISCUSSION TITLE
Sonoff TX T2EU One of the two hardware touch switches is not responding

### DISCUSSION THREAD

**Discussion by ManosAth** [arendst/Tasmota#20363] [Support request]
Hello and Happy new year to everybody

I managed to flash a large number of TX T2EU wall switches successfully (45pcs) but about 8 of them after flashing although they are working from TASMOTA REMOTA or TASMOTROL or on my PC browser,  the left side "hardware" switch is not responding when I am pressing it. Please note that the same switch is working when I am using it to connect an RF remote switch.

Any ideas ?

**Comment by sfromis** [comment]
Touch can be quite finicky, and may depend on physical details of mounting.

**Reply by ManosAth** [reply]
Thank you for your reply. Unfortunately it is not the mounting position because I am testing them also out of their base. The same happens even when I am trying the PCB on my hand.

**Reply by sfromis** [reply]
Out of their base, or in your hand is also mounting details, which can influence the electrical characteristics around touch detection. And what was done during flashing, with soldering and so on is also factors to look at.
